test/doc/affinity.md

```diff
@@ -2,12 +2,12 @@
 
 | Case ID | Title                                                                                               | Priority | Smoke | Status | Other |
 | ------- |-----------------------------------------------------------------------------------------------------| -------- | ----- | ------ | ----- |
-| L00001  | Successfully run deployment/pod who is bound to an IPPool set with matched nodeSelector             | p2       | true  | NA     |       |
-| L00002  | Failed to run deployment/pod who is bound to an IPPool set with no-matched nodeSelector             | p3       |       | NA     |       |
-| L00003  | Successfully run deployment/pod who is bound to an IPPool set with matched namespaceSelector        | p2       | true  | NA     |       |
-| L00004  | Failed to run deployment/pod who is bound to an IPPool set with no-matched namespaceSelector        | p3       |       | NA     |       |
-| L00005  | Successfully run deployment/pod who is bound to an IPPool set with matched podSelector              | p2       | true  | NA     |       |
-| L00006  | Failed to run deployment/pod who is bound to an IPPool set with no-matched podSelector              | p3       |       | NA     |       |
-| L00007  | Successfully run daemonSet/pod who is cross-zone daemonSet with matched nodeSelector                | p2       |       | NA     |       |
-| L00008  | Successfully restarted statefulSet/pod with matching podSelector, ip remains the same               | p2       |       | NA     |       |
+| L00001  | Successfully run deployment/pod who is bound to an IPPool set with matched `NodeAffinity`           | p2       | true  | done   |       |
```

Comment:
```suggestion
| L00001  | Successfully run deployment/pod that is bound to an IPPool set with matched `NodeAffinity`           | p2       | true  | done   |       |
```
If possible, `who` is used for an object with life and `that` for everything. Same below.